x/auth/keeper/account.go

```diff
@@ -42,6 +49,17 @@ func (ak AccountKeeper) GetAccount(ctx sdk.Context, addr sdk.AccAddress) types.A
 	return ak.decodeAccount(bz)
 }
 
+// GetAccountAddressById returns account address by id.
+func (ak AccountKeeper) GetAccountAddressByID(ctx sdk.Context, id uint64) string {
+	store := ctx.KVStore(ak.key)
+	bz := store.Get(types.AccountNumberStoreKey(id))
+	if bz == nil {
+		return ""
+	}
+
+	return conv.UnsafeBytesToStr(bz)
```

Comment:
Hmm, so we're storing `conv.UnsafeBytesToStr(bz)` as the value in the DB, which is the address. But sdk.AccAddress is already bytes, so why not just store the bytes directly, i.e. `addr.Bytes`?